crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -221,11 +221,12 @@ MAP
 ....xxxxxxx...@
 ENDMAP
 
-NAME:  nicolae_shoals_entry_flooding
-TAGS:  shoals_entry patrolling transparent
-KMONS: ON = water nymph
-KFEAT: O = enter_shoals
-SUBST: x = x`, ` = W., N = N .:80
+NAME:   nicolae_shoals_entry_flooding
+TAGS:   shoals_entry patrolling transparent
+KMONS:  ON = water nymph
+KFEAT:  O = enter_shoals
+NSUBST: N = 2=N` / `
```

Comment:
I think I would go further and make this just 1-2 nymphs, and probably at least half weight. It's nasty to fight even one water nymph in Lair, which is fine for a vault, but shoals entries place in 50% of games, so this vault will place a lot. One nymph for 75% of the time and two for 25% with default weight would probably be fine as well. But if you like placing up to three, strongly suggest lowering weight, and it's pretty reasonable to give it lower weight regardless of the nymph count.